### PR TITLE
Add 2018 Fall recruitment section

### PR DESCRIPTION
**Pull request by bfbachmann**
I only changed the `Join Us` section of `index.html` and the css files. The rest of the diff is reformatting.

Preview: https://5b77b79ac6aed62584016d24--ubclaunchpad.netlify.com/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,596 +2,626 @@
...
-          <img src="/img/projects/driver/driver.png" />
-        </div>
-        <div class="blurb">
-          <h3>🚗 Getting everyone home safe</h3>
-        </div>
-        <p class="small">
-          Driver is a ride-sharing app for Android that ensures everyone gets home safely. Users provide their destination address
-          and Driver intelligently matches passengers to drivers to minimize driving time. When passengers are matched to
-          drivers, Driver generates an efficient route to each destination. Driver is built using Java and Firebase.
+    <!-- JOIN -->
```
Comment (bfbachmann) [inline index.html]: These are the only actual changes I've made. The rest is reformatting to 4 spaces indents 😡

Comment (bobheadxi) [reply index.html]: this is gonna cause some insane conflicts with my branch 😢

Comment (bobheadxi) [reply index.html]: could you hold off on the indent changes?

Comment (bfbachmann) [reply index.html]: Sure.

Comment (bfbachmann) [reply index.html]: @bobheadxi Bingo bongo. I guess we should just leave this as 2 space indentation. It honestly doesn't matter.

File: index.html [hunk 2]
```diff
@@ -416,25 +416,43 @@ <h1 class="team">Our Team</h2>
...
-        <a class="button" href="https://goo.gl/forms/VhXR8tI7FZnjP9Q52">APPLY</a>
-      </div>
-    </div> -->
+    <br /> -->
+    <div class="row">
+        <div class="col new-position">
+            <h2>Developer</h2>
+            <br />
+            <p class="small">
+                Join our development team to improve your skills and resume. You have the basics down, but want to collaborate with a team
```
Comment (frostyshadows) [inline index.html]: maybe `but want to collaborate with a team to complete a cool project and and learn professional development practices.`

Comment (frostyshadows) [inline index.html:430]: maybe `but want to collaborate with a team to complete a cool project and and learn professional development practices.`

File: index.html [hunk 3]
```diff
@@ -416,25 +416,43 @@ <h1 class="team">Our Team</h2>
...
+                of a project from conception to release.
+            </p>
+            <br />
+            <a class="button" href="https://goo.gl/forms/HoQxm48Sgo30u1ks1">APPLY</a>
+        </div>
+        <div class="col new-position">
+            <h2>Business</h2>
+            <br />
+            <p class="small">
+                Join our business strategy team. Develop industry relationships and awareness while securing funding and sponsorships.
```
Comment (frostyshadows) [inline index.html]: "insight" would sound better than "awareness".

**Review by frostyshadows — CHANGES_REQUESTED** [review]

**Pull request merged**